redaxo/src/addons/structure/pages/linkmap.php

```diff
@@ -79,6 +79,11 @@ function insertLink(link,name){
 $isRoot = $category_id === 0;
 $category = rex_category::get($category_id);
 
+// Hide minibar (REX 5.7+)
+if (class_exists('rex_minibar')) {
```

Comment:
```suggestion
if (class_exists('rex_minibar') && && rex_minibar::getInstance()->isActive() === null) {
```

damit wir hier nur eine entscheidung treffen, falls vorher nicht schon jemand aktiv true/false gesetzt hat

Reply:
Funktioniert. Im project-AddOn dann:

```php
rex_extension::register('PACKAGES_INCLUDED', function () {
    if (rex_be_controller::getCurrentPagePart(1) == 'linkmap') {
        rex_minibar::getInstance()->setActive(true);
    }
});
```